pkg/build/api/types.go

```diff
@@ -559,10 +556,30 @@ type BuildConfigSpec struct {
 	// are defined, a new build can only occur as a result of an explicit client build creation.
 	Triggers []BuildTriggerPolicy
 
+	// SchedulePolicy describes how the new build created from this build
+	// configuration will be scheduled for execution.
+	// This is optional, if not specified we default to "parallel".
+	SchedulePolicy *BuildSchedulePolicy
```

Comment:
Since this doesn't apply to Builds, it makes sense to me that it is not part of BuildSpec.